### PR TITLE
add ch15

### PR DESCRIPTION
**Pull request by tonyelhabr**

### CODE REVIEW

**Review by czeildi — CHANGES_REQUESTED** [review]
Could you please fix the rendering issue 
> Duplicate chunk label 'setup'
? https://github.com/r4ds/bookclub-tmwr/pull/71/checks

I think these labels should be unique in the whole book not only in a single Rmd file

**Review by czeildi — APPROVED** [review]

**Review by czeildi — DISMISSED** [review]
unfortunately the rendering phase still is not successful. https://github.com/r4ds/bookclub-tmwr/pull/71/checks?check_run_id=2410595709

File: 15-screening_many_models.Rmd [hunk 1]
```diff
@@ -1,13 +1,491 @@
...
+    freedom_to_make_life_choices,
+    generosity,
+    perceptions_of_corruption
+  )
+
+cors <-
+  df_selected %>% 
+  select(where(is.numeric)) %>% 
+  corrr::correlate() %>% 
+  rename(col1 = rowname) %>%
```
Comment (tanho63) [inline 15-screening_many_models.Rmd:82]: Looks like it's a versioning issue for corrr::correlate() 
image

```suggestion
  rename(col1 = term) %>% 
```

**Review by tanho63 — CHANGES_REQUESTED** [review]

File: 15-screening_many_models.Rmd (hunk 1 above)
Comment (tanho63) [reply 15-screening_many_models.Rmd:82]: https://github.com/tidymodels/corrr/issues/117#issuecomment-712460391

**Comment by tanho63** [comment]
@tonyelhabr I am, once again,,,,
https://github.com/r4ds/bookclub-tmwr/pull/71/checks?check_run_id=2411779415#step:8:1780

asking for the rules pkg in description? :P

**Review by tanho63 — APPROVED** [review]

**Pull request merged**